chain/client/src/sync_jobs_actions.rs

```diff
@@ -80,6 +81,19 @@ impl SyncJobsActions {
         Ok(success)
     }
 
+    pub fn handle_load_memtrie_request(&mut self, msg: LoadMemtrieRequest) {
+        let result = msg
```

Comment:
Quick comment here saying this call is synchronous and handled in the sync_jobs_actor (which is fine).

Reply:
Added in b94876d1df345b5d73229769517b5430be300900